test/runtests.jl

```diff
@@ -21,39 +21,43 @@ synthetic = _load_synthetic()
 generators = generator_catalogue
 
 @testset "CounterfactualExplanations.jl" begin
-    @testset "Data" begin
-        include("data/data.jl")
-    end
+    # @testset "Data" begin
+    #     include("data/data.jl")
+    # end
 
-    @testset "Data preprocessing" begin
-        include("data_preprocessing.jl")
-    end
+    # @testset "Data preprocessing" begin
+    #     include("data_preprocessing.jl")
+    # end
 
-    @testset "Generative Models" begin
-        include("generative_models.jl")
-    end
+    # @testset "Generative Models" begin
+    #     include("generative_models.jl")
+    # end
 
-    @testset "Counterfactuals" begin
-        include("counterfactuals.jl")
-    end
+    # @testset "Counterfactuals" begin
+    #     include("counterfactuals.jl")
+    # end
 
-    @testset "Generators" begin
-        include("generators.jl")
-    end
+    # @testset "Generators" begin
+    #     include("generators.jl")
+    # end
 
-    @testset "Probe" begin
-        include("probe.jl")
-    end
+    # @testset "Probe" begin
+    #     include("probe.jl")
+    # end
```

Comment:
you know whats wrong

Reply:
Yes, I'm making some late experiments with additional tests, will fix once the experiment is finished